spec/examples/complete/Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM alpine
+FROM bash
 
-ENTRYPOINT /app/job.sh
+COPY app app
+COPY . .
 
-LABEL com.ngageoint.seed.manifest="SEED_MANIFEST"
```

Comment:
Need the LABEL and SEED_MANIFEST placeholder so that it can be rolled into releases via `inject-manifests.sh` script:

https://github.com/ngageoint/seed/blob/master/spec/examples/inject-manifests.sh